sql/mozfun/norm/diff_months/udf.sql

```diff
@@ -1,32 +1,37 @@
-CREATE OR REPLACE FUNCTION norm.subscription_months_renewed(
+CREATE OR REPLACE FUNCTION norm.diff_months(
   start DATETIME,
   `end` DATETIME,
-  grace_days INT64
+  grace_period INTERVAL,
+  inclusive BOOLEAN
```

Comment:
In renamed this function to be more generic I added the `inclusive` argument to make it clear whether the interval from start to end is considered inclusive (jan 1 to feb 1 returns 1) or exclusive (jan 1 to feb 1 returns 0)